compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/LangLibFunctionBinder.java

```diff
@@ -43,8 +43,8 @@ public class LangLibFunctionBinder {
     private final Types types;
     private final CompilerContext context;
 
-    public LangLibFunctionBinder(Types types, CompilerContext context) {
-        this.types = types;
+    public LangLibFunctionBinder(CompilerContext context) {
```

Comment:
This is a breaking change right? We should deprecate the existing constructor and introduce a new one.  Let's omit this change as this is not related to our changes.